iree/vm/test/BUILD

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//iree/tools:compilation.bzl", "iree_bytecode_module")
+load("//build_tools/bazel:tblgen.bzl", "gentbl")
+load("//build_tools/embed_data:build_defs.bzl", "cc_embed_data")
+
+package(
+    default_visibility = ["//visibility:public"],
+    licenses = ["notice"],  # Apache 2.0
+)
+
+cc_embed_data(
+    name = "all_bytecode_modules_cc",
+    srcs = [
+      ":arithmetic_ops.module",
+      ":control_flow_ops.module",
+    ],
+    cc_file_output = "all_bytecode_modules.cc",
+    cpp_namespace = "iree::vm::test",
+    flatten = True,
+    h_file_output = "all_bytecode_modules.h",
+)
+
+iree_bytecode_module(
+    name = "arithmetic_ops",
+    src = "arithmetic_ops.mlir",
+    cc_namespace = "iree::vm::test",
```

Comment:
This isn't actually used, right? You're only using `all_bytecode_modules_cc`. I really don't like the iree_bytecode_module magic cc_embed data integration, so the less we do of that, the better IMO (until I can clean it up and make those all proper rules).

Reply:
this is used - it's how the file is generated for all_bytecode_modules_cc

Reply:
I meant specifically the `cc_namespace`. I'll send a fix.